client/my-sites/plan-features/style.scss

```diff
@@ -230,13 +238,23 @@ $plan-features-header-banner-height: 20px;
 	margin: 0;
 	font-size: 14px;
 	color: $gray;
+	position: relative;
 
 	&.is-placeholder {
 		@include placeholder( 23% );
 		height: 50px;
 		margin-top: 5px;
 	}
 }
+.plan-features__footer-desc-longest {
+	visibility: hidden;
```

Comment:
Could I get a design review of how terrible of a hack this is? Maybe @rickybanister or @shaunandrews ?